doc/changes/latest.inc

```diff
@@ -70,6 +70,9 @@ Enhancements
 
 Bugs
 ~~~~
+
+- Pick also bad channels when selecting T1T2 mag indices (:gh:`10639` by :newcontrib:`Matti Toivonen`_)
```

Comment:
Might as well give a hint where people will see the change while fixing the ref bug
```suggestion
- Pick also bad channels when selecting T1T2 magnetometers in :func:`mne.preprocessing.maxwell_filter` (:gh:`10639` by :newcontrib:`Matti Toivonen`)
```